src/main/resources/layouts/settings.fxml

```diff
@@ -414,7 +414,7 @@
                                           </HBox>
                                           <HBox prefHeight="42.0" prefWidth="366.0">
                                              <children>
-                                                <Label text="Export your current Data to an Sql script. This is needed if you go from the Snapshot-1.2.0-KeepTimeto the recent release version. " wrapText="true" />
+                                                <Label text="Export your current Data to an Sql script. This is needed if you upgrade to an newer KeepTime version. " wrapText="true" />
                                              </children>
```

Comment:
Could you also check the icon for the export done confirmation?
![image](https://user-images.githubusercontent.com/5772511/212905716-13338407-631c-4125-904b-92b8afeaca47.png)
(not related to the issue, but would fit)

Reply:
![grafik](https://user-images.githubusercontent.com/86607279/213702911-831f0158-4162-46c3-b450-a41d8eb7c67f.png)
Added it :)